guides/doc-Provisioning_Guide/topics/Introduction.adoc

```diff
@@ -49,8 +68,8 @@ If you use the Discovery service, {Project} automatically detects the MAC addres
 
 *PXE-less Provisioning*
 
-After you submit a new host request, you must boot the specific host with a boot disk that you download from {Project}.
+After you submit a new host request, you must boot the specific host with a boot disk that you download from {Project} and transfer and attach to a USB port of the host.
 
 *Compute Resources*
 
-The compute resource creates the virtual machine for the host and returns the MAC address to {Project}. If you use image-based provisioning, the host does not follow the standard PXE boot and operating system installation: the compute resource creates a copy of the image for the new host to use.
+The {Project} creates the virtual machine for the host and returns the MAC address to {Project}. If you use image-based provisioning, the host does not follow the standard PXE boot and operating system installation: the compute resource creates a copy of the image for the new host to use. Depending on image settings in {Project}, seed data can be passed in for initial configuration (e.g. via cloud-init) or {Project} can perform SSH connection to the server and execute a template to finish off customization.
```

Comment:
Can we leave out the seed data cloud-init stuff from here because that is specific information supported for certain use cases and not general enough for the general introduction to the project.

Reply:
Well if you look above it says Compute Resources and cloud-init is essential in cloud and also in virtualized environments. I can move somewhere else if you insist but the old content was not explaining much.

Reply:
The first sentence sounds odd. Project doesn't return MAC to Project, it probably fetches the MAC

Reply:
Can you explain the separation of virtual machine and host in this se

> The first sentence sounds odd. Project doesn't return MAC to Project, it probably fetches the MAC

Can someone explain to me the separation of virtual machine and host in this same sentence also?

Reply:
Yeah it should be: The {Project} creates the virtual machine for the host and stores the MAC address returned by the hypervisor into the {Project}.

Or better into Foreman's database (we like to call it an inventory as well).